docs/templates/preprocessing/image.md

```diff
@@ -28,6 +28,10 @@ Generate batches of tensor image data with real-time data augmentation. The data
     - __width_shift_range__: Float (fraction of total width). Range for random horizontal shifts.
     - __height_shift_range__: Float (fraction of total height). Range for random vertical shifts.
     - __shear_range__: Float. Shear Intensity (Shear angle in counter-clockwise direction as radians)
+    - __zoom_range__: Float or [lower, upper]. Range for random zoom.
```

Comment:
Shouldn't we explain if zoom_range is float `[lower, upper]` is `[1-zoom_range, 1+ zoom_range]`, because it is not apparent from the documentation.